src/assets/styles/article.css

```diff
@@ -0,0 +1,52 @@
+#article-body {
```

Comment:
I'm pretty sure everything in this css file can be done using `useStyles` and should be implemented in that way so navigating the styles is easier (goes back to your point about maintainability).

Reply:
The main article section has a designated class called `main` anyway where you can put all of this CSS instead of adding the `article-body` id and adding styles under that.

Reply:
Writing plain CSS is easier than `useStyles` boilerplate imo. I have a feeling it's more efficient on older browsers too. For maintainability we should move everything to CSS files because there's no reason to mix it with javascript because we don't really need anything from JS... are there benefits?